src/api/app/models/update_notification_events.rb

```diff
@@ -14,6 +14,10 @@ def create_events
         e.elements('data') do |d|
           data[d['key']] = d['_content']
         end
+        puts type
```

Comment:
:eyes:

Reply:
All this stuff is in a WIP commit. The description says that it's blocked for now...

Reply:
I didn't want to forget this when working on it this week. This is why I commited it.